code/parse/sexp.cpp

```diff
@@ -32844,6 +32889,12 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"\tRest: The player start ship to operate on.\r\n"
 		"\tOnly really useful for multiplayer."},
 
+	{ OP_SET_HOTKEY, "set-hotkey\r\n"
+		"\tSets hotkey for the specified ship(s) or wing(s)\r\n"
```

Comment:
It should probably mention that, if the ship(s) are already in the hotkey set, they'll be removed.

Reply:
Ah good point, though in my tests the running the test again did not remove the current hotkeys set by the player or the mission. To actually clear the hotkeys I would have to use the function `hud_target_hotkey_clear()`

Also, in my tests running the sexp with the same arguments 4 seconds apart did not remove the ships in the hotkey set. So for example set-hotkey 1 for Beta did not remove Beta wing from hotkey 1 on the second evaluation of the sexp.

Reply:
Ah, update. So if I use `HOTKEY_USER_ADDED` then it does remove them, but if I use `HOTKEY_MISSION_FILE_ADDED` the sexp is ignored since they are already part of that hotkey set. I will work on the logic of this a submit a revision.